npm-shrinkwrap.json

```diff
@@ -1840,7 +1840,7 @@
         },
```

Comment:
Let's drop shrinkwrap file from the changeset since `package.json` wasn't modified either.